doc/en/fixture.rst

```diff
@@ -879,7 +879,7 @@ Here's what that might look like:
         admin_client.delete_user(user)
 
 
-    def test_email_received(receiving_user, email):
+    def test_email_received(sending_user, receiving_user, email):
         email = Email(subject="Hey!", body="How's it going?")
         sending_user.send_email(_email, receiving_user)
```

Comment:
Unrelated to your fix, but I think this should also be

```suggestion
        sending_user.send_email(email, receiving_user)
```

for this to actually work, no?

Reply:
Yes, should I commit your suggestion?

Reply:
i beleive a key issue of why this example bit-rotted/was incorrect to begin with, is that its not part of the regendoc process (where we update the example outputs in the docs by running the examples)

as  a followup/extension we might want to add a actual testrun of this stuff

Reply:
> Yes, should I commit your suggestion?

In general yes, but I did take the liberty of going ahead and doing it myself, thanks. 👍

Reply:
> is that its not part of the regendoc process 

I agree, created https://github.com/pytest-dev/pytest/issues/8213. 👍